Data/Context/EcommerceContext.cs

```diff
@@ -0,0 +1,101 @@
+﻿using EFCoreConsoleApp.Data.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace EFCoreConsoleApp.Data.Context
```

Comment:
Please add meaningful PR name and description. Also rename the entire application to EcommerceStore or smth similar as this app will be modified and converted to web app in future stages